src/app/components/community/sidebar/sidebar.ts

```diff
@@ -49,18 +49,37 @@ export default class AppCommunitySidebar extends Vue {
 	@Prop(Array)
 	collaborators!: User[];
 
-	@Prop(Boolean)
-	hasMoreCollaborators!: boolean;
+	@Prop(Number)
+	collaboratorCount!: number;
 
 	@Prop(Number)
 	initialCollaboratorCount!: number;
 
 	@State
 	app!: Store['app'];
 
-	collaboratorListCollapsed = false;
+	currentCollaborators: User[] = [];
+	currentCollaboratorCount = 0;
+
+	collaboratorListCollapsed = true;
 	isLoadingMoreCollaborators = false;
 	isShowingShare = false;
+	loadedAll = false;
+
+	readonly GJ_IS_CLIENT = GJ_IS_CLIENT;
+
+	@Emit('collaborators')
+	emitCollaborators(_collaborators: User[]) {}
```

Comment:
The emit doesn't seem to be used by anything?

Reply:
Yeah, we'll reimplement once needed.